data/tmdb-movies/src/main/java/io/fimltime/data/tmdb/movies/MovieDetailEntityExt.kt

```diff
@@ -0,0 +1,38 @@
+package io.fimltime.data.tmdb.movies
+
+import io.filmtime.data.database.MovieDetailEntity
+import io.filmtime.data.model.VideoDetail
+import io.filmtime.data.model.VideoId
+
+fun MovieDetailEntity.toMovie(): VideoDetail {
+  return VideoDetail(
+    ids = VideoId(tmdbId = tmdbId, traktId = null),
+    title = name,
+    posterUrl = posterUrl,
+    coverUrl = coverUrl,
+    year = year,
+    genres = listOf(),
```

Comment:
if we want to really work offline we have to handle `genres` and `spokenLanguages` as well. If you do not to do it now, please add a TODO with an issue number